src/main/java/net/minecraftforge/fml/client/GuiSlotModList.java

```diff
@@ -86,20 +93,24 @@ protected void drawSlot(int idx, int right, int top, int height, Tessellator tes
         if (Loader.instance().getModState(mc) == ModState.DISABLED)
         {
             font.drawString(font.trimStringToWidth(name,       listWidth - 10), this.left + 3 , top +  2, 0xFF2222);
-            font.drawString(font.trimStringToWidth(version,    listWidth - 10), this.left + 3 , top + 12, 0xFF2222);
+            font.drawString(font.trimStringToWidth(version,    listWidth - (4 + height)), this.left + 3 , top + 12, 0xFF2222);
             font.drawString(font.trimStringToWidth("DISABLED", listWidth - 10), this.left + 3 , top + 22, 0xFF2222);
         }
         else
         {
             font.drawString(font.trimStringToWidth(name,    listWidth - 10), this.left + 3 , top +  2, 0xFFFFFF);
-            font.drawString(font.trimStringToWidth(version, listWidth - 10), this.left + 3 , top + 12, 0xCCCCCC);
+            font.drawString(font.trimStringToWidth(version, listWidth - (4 + height)), this.left + 3 , top + 12, 0xCCCCCC);
             font.drawString(font.trimStringToWidth(mc.getMetadata() != null ? mc.getMetadata().getChildModCountString() : "Metadata not found", listWidth - 10), this.left + 3 , top + 22, 0xCCCCCC);
 
-            switch(vercheck.status) //TODO: Change to icons?
+            switch(vercheck.status)
             {
                 case BETA_OUTDATED:
                 case OUTDATED:
-                    font.drawString("U", right - font.getCharWidth('U') - 1, top+height-font.FONT_HEIGHT+2, 0x22FF22);
+                    Minecraft.getMinecraft().getTextureManager().bindTexture(NOTIFICATION_ICON);
+                    GlStateManager.color(1, 1, 1, 1);
+                    GlStateManager.pushMatrix();
+                    Gui.drawModalRectWithCustomSizedTexture(right - (8 + height / 2 - 4), top + height / 2 - 4, 0, ((System.currentTimeMillis() / 800 & 1) == 1) ? 8 : 0, 8, 8, 8, 16);
+                    GlStateManager.popMatrix();
```

Comment:
The eventual plan was to have multiple icons or whatever of displaying each of these states. But not sure what ones people care about being different.